adafruit_esp32spi/adafruit_esp32spi.py

```diff
@@ -766,6 +766,8 @@ def get_time(self):
         """The current unix timestamp"""
         if self.status == WL_CONNECTED:
             resp = self._send_command_get_response(_GET_TIME)
+            if struct.unpack('<i', resp[0]) == (0,):
```

Comment:
unpack it once, then compare the unpacked val to 0 or return

Reply:
Of course!